synth/generate

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Define an array of collections
+declare -a collections=("create_namespace" "activity_exists" "set_activity_attributes"
```

Comment:
you don't need to "declare" an array like this, when you are immediately assigning
```
collections=("create_namespace" "activity_exists" "set_activity_attributes" blah blah)
```

Reply:
Actually looking a bit closer - if you separated the json files into a separate directory - you probably don't even need this array since you can get the same info from an `ls`

Reply:
I've still had to keep a `to_exclude` collection of json input files because I can't get Synth to play nicely when I move the input files into a subdirectory. Cannot figure it out. Things seem to compile and generate something, but the format once in Chronicle causes a parse error. The syntax I was using to reference the file was `"sub_directory.@input_json"`, as opposed to `"@input_json"` as I have it now with everything in one folder.